datasets/PV26_utrecht/PV26_utrecht.derived.ad

```diff
@@ -126,37 +104,59 @@
+- heat_households_indoor_pipelines_investment_costs_without_block_heating_eur = 5984.0
+- heat_buildings_indoor_investment_costs_eur_per_kw = 151.0
+- heat_buildings_heat_meter_investment_costs_eur_per_kw = 1.35
+- heat_buildings_heat_meter_investment_costs_eur_per_connection = 913.0
+- heat_exchanger_station_investment_costs_eur_per_kw = 80.0
+- heat_sub_station_investment_costs_eur_per_kw = 135.0
+- heat_distribution_pipelines_investment_costs_eur_per_meter = 1200.0
+- heat_length_of_distribution_pipelines_in_meter_per_residence_object_first_bracket = 4.263307
+- heat_length_of_distribution_pipelines_in_meter_per_residence_object_second_bracket = 5.105242
+- heat_length_of_distribution_pipelines_in_meter_per_residence_object_third_bracket = 5.706534
+- heat_length_of_distribution_pipelines_in_meter_per_residence_object_fourth_bracket = 6.389874
+- heat_length_of_distribution_pipelines_in_meter_per_residence_object_fifth_bracket = 8.392647
+- heat_length_of_connection_pipelines_in_meter_per_residence_first_bracket = 3.267976
+- heat_length_of_connection_pipelines_in_meter_per_residence_second_bracket = 4.354065
+- heat_length_of_connection_pipelines_in_meter_per_residence_third_bracket = 5.256497
+- heat_length_of_connection_pipelines_in_meter_per_residence_fourth_bracket = 6.115599
+- heat_length_of_connection_pipelines_in_meter_per_residence_fifth_bracket = 7.953715
+- heat_primary_pipelines_investment_costs_per_kw = 200.0
+- heat_yearly_indoor_infrastructure_maintenance_costs_factor = 0.025
+- heat_yearly_outdoor_infrastructure_maintenance_costs_factor = 0.03
 - electric_vehicle_profile_1_share = 1.0
 - electric_vehicle_profile_2_share = 0.0
 - electric_vehicle_profile_3_share = 0.0
 - electric_vehicle_profile_4_share = 0.0
 - electric_vehicle_profile_5_share = 0.0
 - solar_pv_profile_1_share = 1.0
 - solar_pv_profile_2_share = 0.0
-- annual_infrastructure_cost_gas = 104.88472639037296
-- areable_land = 809.94
-- co2_emission_1990 = 6.593246
+- annual_infrastructure_cost_gas = 109.4606147314352
+- areable_land = 902.21
+- co2_emission_1990 = 7.074759
 - coast_line = 0.624
 - interconnector_capacity = 60000.0
-- total_land_area = 1380.28
-- number_of_buildings = 71198.0
-- number_of_cars = 642013.0
-- number_of_residences = 548114.0
-- number_of_inhabitants = 1273613.0
-- offshore_suitable_for_wind = 3719.697785
-- residences_roof_surface_available_for_pv = 12.512592
-- buildings_roof_surface_available_for_pv = 7.830389
-- other_emission_agriculture = 0.884841
-- other_emission_built_environment = 0.044547
-- other_emission_transport = 0.113467
-- other_emission_industry_energy = 0.179275
-- number_of_detached_houses = 40056.719234
-- number_of_apartments = 198006.1825
-- number_of_semi_detached_houses = 41169.938768
-- number_of_corner_houses = 72187.710028
-- number_of_terraced_houses = 196693.44947
-- base_dataset = nl
+- total_land_area = 1487.52
+- number_of_buildings = 37084.0
```

Comment:
Wow, it seemed as if the number of buildings has dropped by half. Maybe we should double check that number based on Klimaatmonitor/CBS data for the province?